pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -1333,6 +1333,15 @@ var _ = Describe("Converter", func() {
 			Expect(domain.Spec.Devices.Inputs[0].Bus).To(Equal("usb"), "Expect usb bus")
 		})
 
+		It("should enable usbredir when numberOfDevices > 0", func() {
```

Comment:
Could you please also add tests for when numberOfDevices is negative and zero?

Reply:
I've added those, thanks!